.travis.yml

```diff
@@ -0,0 +1,42 @@
+sudo: false
+
+os: osx
+osx_image: xcode9.2
+compiler:
+  - clang
+script:
+  - brew tap aws/tap
+  - brew install --build-bottle aws-sam-cli
+  - export SAM_CLI_VERSION=$(cat $TRAVIS_BUILD_DIR/Formula/aws-sam-cli.rb | grep -v root_url | grep url | awk -F'/v' '{print $2}' | awk -F '.tar' '{print $1}')
+  - export ROOT_URL=$(echo "https://github.com/aws/homebrew-tap/releases/download/v"$SAM_CLI_VERSION"/")
```

Comment:
what is the purpose of this? We don't do any release on homebrew-tap repo.

Reply:
my thinking was to move hosting of bottles here eventually.